### PR TITLE
Initial setup for the terra-toolkit-docs package.

### PR DESCRIPTION
**Pull request by mjhenkes**
### Summary

This PR sets up the placeholder package for terra-toolkit-docs. It's mainly here to help the next PR be easier to consume.

### Additional Details

@cerner/terra

[CONTRIBUTORS.md]: ../blob/main/CONTRIBUTORS.md
[License]: ../blob/main/LICENSE

### CODE REVIEW

File: packages/terra-toolkit-docs/.npmignore [hunk 1]
```diff
@@ -0,0 +1,8 @@
+*.log
```
Comment (emilyrohrbough) [inline packages/terra-toolkit-docs/.npmignore:1]: Thoughts on migrating to the files package.json key for defining what should be released? It seems we sometimes end up with more in the release than needed with this pattern

File: packages/terra-toolkit-docs/README.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+## Versioning
+
+terra-toolkit-docs is considered to be stable and will follow [SemVer](http://semver.org/) for versioning.
+
+1. MAJOR versions represent breaking changes
+2. MINOR versions represent added functionality in a backwards-compatible manner
+3. PATCH versions represent backwards-compatible bug fixes
+
+Consult the component CHANGELOGs, related issues, and PRs for more information.
```
Comment (emilyrohrbough) [inline packages/terra-toolkit-docs/README.md:16]: this can be removed right?

File: packages/terra-toolkit-docs/README.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
+# Eslint Config Terra
```
Comment (emilyrohrbough) [inline packages/terra-toolkit-docs/README.md]:
```suggestion
# Terra Toolkit Docs
```

File: packages/terra-toolkit-docs/package.json [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    "Cerner",
+    "Terra"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-toolkit/issues"
+  },
+  "engines": {
+    "node": "^10.13.0 || >=11.10.1"
```
Comment (emilyrohrbough) [inline packages/terra-toolkit-docs/package.json]: What is blocking this from being on the latest? wasn't it wdio?

**Review by emilyrohrbough — APPROVED** [review]

File: packages/terra-toolkit-docs/.npmignore (hunk 1 above)
Comment (mjhenkes) [reply packages/terra-toolkit-docs/.npmignore:1]: I think generally it's safer to publish more than publish a version where someone forgot to include a file? 🤷‍♂️

File: packages/terra-toolkit-docs/README.md (hunk 2 above)
Comment (mjhenkes) [reply packages/terra-toolkit-docs/README.md:16]: it's still useful, for example: https://www.npmjs.com/package/@cerner/eslint-config-terra#versioning

File: packages/terra-toolkit-docs/package.json (hunk 4 above)
Comment (mjhenkes) [reply packages/terra-toolkit-docs/package.json]: Yes, i'll update it to just a have the minimum version.

**Review by benbcai — APPROVED** [review]

**Pull request merged**